packages/unmock-core/src/interfaces.ts

```diff
@@ -1,8 +1,7 @@
-import { UnmockOptions } from "./options";
 import { IService } from "./service/interfaces";
 
 export interface ILogger {
-  log: (message: string) => void;
+  log(message: string): void;
```

Comment:
Touched these so that IDE-wise they show up as methods and not "callable properties".